src/Commands/EnvCreateCommand.php

```diff
@@ -55,7 +57,7 @@ public function createBuildEnv(
         if ('dev' === $multidev) {
             $this->log()->notice('dev has been passed to the multidev option. Reverting to dev:env:push as dev is not a multidev environment.');
             // Run build:env:push.
-            $this->pushCodeToPantheon($site_env_id, $multidev, '', $env_label);
+            $this->pushCodeToPantheon($site_env_id, $multidev, '', $env_label, $options['message'], $options['git-force']);
```

Comment:
@greg-1-anderson @sugaroverflow,it looks like this was a bug. This addition uses `git-force` when the param is actually `no-git-force`

Reply:
The right way to do this with the annotated command library is to use `'git-force' => true`. That will make `$options['git-force']` true by default, and will add a `--no-git-force` option to set it to false.

Reply:
ok, so #361 needs more work to accommodate that then?

Reply:
You're right, I didn't notice the double-negative of $noGitForce in pushCodeToPantheon(). Added a `!` to the PR.